modules/ossm-auto-route.adoc

```diff
@@ -64,7 +64,7 @@ Then, the following OpenShift Routes are created automatically. You can check th
 
```

Comment:
in 2.0 IOR spec path has been changed and it is enabled true by default in 2.0 SMCP. 
"Enabling Automatic Route Creation" looks out of date. @jwendell  may help a review of this section.

Reply:
> in 2.0 IOR spec path has been changed and it is enabled true by default in 2.0 SMCP.
> "Enabling Automatic Route Creation" looks out of date. @jwendell may help a review of this section.

Where's that section? I can't find it in the deploy preview...

Reply:
@jwendell We've moved those to this new topic. https://ossmnewsmcp--ocpdocs.netlify.app/openshift-enterprise/latest/service_mesh/v2x/ossm-custom-resources.html#ossm-auto-route_ossm-custom-resources-v2x